src/Microsoft.Performance.SDK.Runtime.Tests/Discovery/IsolatedAssemblyLoadingTests.cs

```diff
@@ -57,6 +57,40 @@ public void Cleanup()
             }
         }
 
+        [TestMethod]
+        [UnitTest]
+        public void AlwaysSharedAssembliesContainSDKAssemblies()
+        {
+            var sharedAssemblies = IsolationAssemblyLoader.GetAlwaysSharedAssemblies();
+            var assemblyNames = sharedAssemblies.Values.ToHashSet(StringComparer.OrdinalIgnoreCase);
+
+            Assert.IsTrue(assemblyNames.Contains("Microsoft.Performance.SDK.Runtime.NetCoreApp"));
+            Assert.IsTrue(assemblyNames.Contains("Microsoft.Performance.SDK.Runtime"));
+            Assert.IsTrue(assemblyNames.Contains("Microsoft.Performance.SDK"));
+        }
+
+        [TestMethod]
+        [UnitTest]
+        public void AlwaysSharedAssembliesContainReferencedAssemblies()
+        {
+            // Nuget.Versioning is currently referenced, so using it as a test case.
```

Comment:
would it make sense to shim GetReferencedAssemblies?

Reply:
I don't think it's worth it for this considering we'll be using nuget.versioning for semantic versioning for the foreseeable future. If we run into problems, then I can go ahead an over engineer this. :)